api/client/client.go

```diff
@@ -274,6 +273,59 @@ func (c *Client) NewCollection(ctx context.Context, dbID thread.ID, config db.Co
 	return err
 }
 
+// UpdateCollection updates an existing collection.
+func (c *Client) UpdateCollection(ctx context.Context, dbID thread.ID, config db.CollectionConfig, opts ...db.ManagedOption) error {
+	args := &db.ManagedOptions{}
+	for _, opt := range opts {
+		opt(args)
+	}
+	cc, err := collectionConfigToPb(config)
+	if err != nil {
+		return err
+	}
+	_, err = c.c.UpdateCollection(ctx, &pb.UpdateCollectionRequest{
+		DbID:   dbID.Bytes(),
+		Config: cc,
+	})
+	return err
+}
+
+// DeleteCollection deletes a collection.
+func (c *Client) DeleteCollection(ctx context.Context, dbID thread.ID, name string, opts ...db.ManagedOption) error {
+	args := &db.ManagedOptions{}
+	for _, opt := range opts {
+		opt(args)
+	}
+	_, err := c.c.DeleteCollection(ctx, &pb.DeleteCollectionRequest{
+		DbID: dbID.Bytes(),
+		Name: name,
+	})
+	return err
+}
+
+// GetCollectionIndexes returns an existing collection's indexes.
+func (c *Client) GetCollectionIndexes(ctx context.Context, dbID thread.ID, name string, opts ...db.ManagedOption) ([]db.Index, error) {
```

Comment:
Ok, so we'll want to expose this on the remote clients as well then huh?

Reply:
Yep yep